spec/plugins/data_table/general.js

```diff
@@ -559,13 +559,91 @@ describe("DataTable Plugin", function() {
       expect(reorderableSpy).not.toHaveBeenCalled();
     });
 
+    it("should default column resize to off", function() {
+      app.view.dataTable.row("abc", {
+        columns : [
+          { attr: "name", title: "Name" },
+          { attr: "age", title: "Age" }
+        ]
+      });
+
+      app.view.dataTable("def", {
+        rowClassName : "abc"
+      });
+
+      var table = new app.Views.def({ collection : collection }).render();
+
+      expect(table.resizableColumns).toEqual(false, "Table resizableColumns default");
+      expect(table._colReorder.s.allowResize).toEqual(false, "ColReorder allowResize");
+      expect(table._colReorder.s.allowReorder).toEqual(true, "ColReorder allowReorder");
+    });
+
+    it("should allow columns to be resized with flag", function() {
+      app.view.dataTable.row("abc", {
+        columns : [
+          { attr: "name", title: "Name" },
+          { attr: "age", title: "Age" }
+        ]
+      });
+
+      app.view.dataTable("def", {
+        rowClassName : "abc",
+        resizableColumns : true
+      });
+
+      var table = new app.Views.def({ collection : collection }).render();
+
+      expect(table.resizableColumns).toEqual(true, "Table resizableColumns default");
+      expect(table._colReorder.s.allowResize).toEqual(true, "ColReorder allowResize");
+      expect(table._colReorder.s.allowReorder).toEqual(true, "ColReorder allowReorder");
+
+      // these should set to off as they cause display issues
+      expect(table._colReorder.s.bAddFixed).toEqual(false, "ColReorder bAddFixed");
+      expect(table._colReorder.s.bResizeTableWrapper).toEqual(false, "ColReorder bResizeTableWrapper");
+      expect(table._colReorder.s.allowHeaderDoubleClick).toEqual(false, "ColReorder allowHeaderDoubleClick");
+    });
+
+    it('should add dataTable-resizeableColumns class to resizable tables', function() {
```

Comment:
I think it's worth addressing these hound comments about double & single mixed.  We have used double throughout this code base